specification/reasonable-adjustment-flag.yaml

```diff
@@ -105,6 +105,16 @@ info:
 
     For more details on sandbox testing, or to try out the sandbox using our \"Try this API\" feature, see the documentation for each endpoint.
 
+    ### Integration testing
+    Our [integration test environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#integration-testing):
+    * is for formal integration testing
+    * is stateful, so persists updates
+    * includes authorisation, with options for user-restricted access (with or without smartcard) and application-restricted access
+
+    To test updating patient details, you must set up your own test data.
```

Comment:
Can you check with Gurdeep if this is correct?

Reply:
@gjugpal FYI

Reply:
Yes this approach was agreed to prevent people from updating the 'shared' patients